mangaki/mangaki/admin.py

```diff
@@ -8,13 +8,126 @@
     Work, TaggedWork, WorkTitle, Genre, Track, Tag, Artist, Studio, Editor, Rating, Page,
     Suggestion, SearchIssue, Announcement, Recommendation, Pairing, Reference, Top, Ranking,
     Role, Staff, FAQTheme,
-    FAQEntry, ColdStartRating, Trope, Language
+    FAQEntry, ColdStartRating, Trope, Language, WorkCluster
 )
 from mangaki.utils.anidb import AniDB
 from mangaki.utils.db import get_potential_posters
 
 from collections import defaultdict
 from enum import Enum
+from datetime import datetime
+
+
+@transaction.atomic  # In case trouble happens
+def merge_works(request, works_to_merge, admin_queryset, from_cluster):
```

Comment:
Could we have some documentation on the parameters though (in particular, what is `adin_queryset`?)